### DISCUSSION TITLE
notcurses chat room

### DISCUSSION THREAD

**Discussion by joseluis** [dankamongmen/notcurses#1556] [Ideas]
> I'm thinking about launching a Notcurses chat room that people
> can come to with questions or whatnot. I would prefer to have a
> single place. What would people recommend? Options I'm
> considering (I'm happy to look beyond this list) would be:
> 
> - Matrix
> - Discord
> - IRC

Heh, I was also thinking I missed having some kind of live chat for quick questions and such.

I vote to use Discord all the way. It has the most penetration and least friction for new people, and for us too. A lot of people use it and it's like the standard now. I'm in a lot of discord groups and it's always been a breeze. Both the web client and desktop clients works super well.

**Comment by dankamongmen** [comment]
José Luis Cruz left as an exercise for the reader:
> I vote to use Discord all the way. It has the most penetration and least friction for new people, and for us too. A lot of people use it and it's like the standard now. I'm in a lot of discord groups and it's always been a breeze. Both the web client and desktop clients works super well. 

isn't discord the closed sources though? fedora went with matrix
for this reason, if i recall.

what if our chat room was just commits and a github listener
that threw a notification onto your desktop mmmmmm

-- 
nick black -=- https://www.nick-black.com
to make an apple pie from scratch,
you need first invent a universe.

**Reply by joseluis** [reply]
discord is closed source yes.

I have no experience with matrix, but we can try it first privately and see if it works.

> what if our chat room was just commits and a github listener that threw a notification onto your desktop

I think that would be very cumbersome

**Comment by dankamongmen** [comment]
i've got a matrix room. there's surely a better link than this, but try https://app.element.io/#/room/!gShTphhdXbPEwMeIFB:matrix.org

**Comment by tomek-szczesny** [comment]
I think the current "de facto standard" chat service in open source community is libera.chat.
